--- a/src/Input.cpp
+++ b/src/Input.cpp
@@ -88,12 +88,12 @@ bool CHyprspaceWidget::axisEvent(double delta, Vector2D coords) {
     // otherwise, scroll to switch active workspace
     else {
         if (delta < 0) {
-            SWorkspaceIDName wsIDName = getWorkspaceIDNameFromString("r-1");
+            SWorkspaceIDName wsIDName = getWorkspaceIDNameFromString("m-1");
             if (g_pCompositor->getWorkspaceByID(wsIDName.id) == nullptr) g_pCompositor->createNewWorkspace(wsIDName.id, ownerID);
             getOwner()->changeWorkspace(wsIDName.id);
         }
         else {
-            SWorkspaceIDName wsIDName = getWorkspaceIDNameFromString("r+1");
+            SWorkspaceIDName wsIDName = getWorkspaceIDNameFromString("m+1");
             if (g_pCompositor->getWorkspaceByID(wsIDName.id) == nullptr) g_pCompositor->createNewWorkspace(wsIDName.id, ownerID);
             getOwner()->changeWorkspace(wsIDName.id);
         }
